scripts/delayed.js

```diff
@@ -106,6 +106,53 @@ function loadBottomAd() {
   }
 }
 
+function makeSticky() {
+  const adWrappers = document.querySelectorAll('.ad-wrapper');
+  const lastAd = adWrappers[adWrappers.length - 1];
+
+  const bannerWrappers = document.querySelectorAll('.banner-wrapper');
+  const lastBanner = bannerWrappers[bannerWrappers.length - 1];
```

Comment:
can't we do something like 

```
  const lastAd = document.querySelector('.ad-wrapper:last-of-type');
  const lastBanner = document.querySelector('.banner-wrapper:last-of-type');
```

Reply:
no, this didnt work, which is the reason I opted for this